CRM/Geocoder/Upgrader.php

```diff
@@ -29,6 +29,18 @@ public function upgrade_1100() {
     return TRUE;
   }
 
+	/**
+	 *  Update the URL for Open Street Map end point
+	 *
+	 * @throws \CiviCRM_API3_Exception
+	 */
+	public function upgrade_1200() {
+		$this->ctx->log->info( 'Applying update 1200: Update the URL for Open Street Map end point' );
+		CRM_Core_DAO::executeQuery( "UPDATE `civicrm_geocoder` SET `url` = 'https://nominatim.openstreetmap.org' WHERE `url` LIKE '%nominatim.openstreetmap.org%'" );
```

Comment:
Good idea to also catch those who might have been using nominatim.openstreetmap.org via HTTP, not HTTPS (although I would not have thought that anyone might). However, I would make the pattern slightly more specific to avoid hitting anyone using an alternative server where hypothetically the URL path might contain the string nominatim.openstreetmap.org, so I would add “://”: `%nominatim.openstreetmap.org%`

Reply:
What alternative servers?

Reply:
It is possible to run one’s own Nominatim instance.